test/sharness/t0183-namesys-pubsub.sh

```diff
@@ -26,9 +26,11 @@ test_expect_success 'check namesys pubsub state' '
     test_cmp expected state2
 '
 
+# These commands are *expected* to fail. We haven't published anything yet.
 test_expect_success 'subscribe nodes to the publisher topic' '
-    ipfsi 1 name resolve /ipns/$PEERID_0 &&
-    ipfsi 2 name resolve /ipns/$PEERID_0
+    ipfsi 1 name resolve /ipns/$PEERID_0;
+    ipfsi 2 name resolve /ipns/$PEERID_0;
+    true
```

Comment:
@vyzo does this change make sense?

Reply:
hrm, shouldn't we be testing if they succeeded?
that's why i had the `&&`.

Reply:
the assumption is that name resolve returns 0 if it succeeds.

Reply:
Yeah but shouldn't it *not* succeed? Nothing has been published yet, as far as I know.

Reply:
yeah, good point! these are executed for their side-effect of subscribing to the topic.

Reply:
So yes, change makes sense.